portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,51 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
+
+<head>
```

Comment:
Could you check the indent from this line?
Ideally the un-changed line should remain white in this diff view instead of showing the same lines deleted and added back.

Reply:
my cloud shell auto-formats to this indentation, so hopefully it will be consistent from here on out!